.github/workflows/release.yml

```diff
@@ -33,53 +26,54 @@ jobs:
       contents: write
       packages: write
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           ref: ${{ github.event.inputs.release-branch }}
           fetch-depth: 0
           persist-credentials: false
+
       - name: Setup Java and Scala
         uses: olafurpg/setup-scala@v14
         with:
           java-version: openjdk@1.17
+
       - name: Setup Node.js
         uses: actions/setup-node@v3
         with:
           node-version: "lts/*"
-      - name: Setup Helm
-        uses: azure/setup-helm@v3
-        with:
-          version: "3.12.2" # default is latest (stable)
-        id: install
-      - name: Setup yq - portable yaml processor
-        uses: mikefarah/yq@v4.34.2
+
       - uses: crazy-max/ghaction-import-gpg@v3
         id: import_gpg
         with:
           gpg-private-key: ${{ secrets.HYP_BOT_GPG_PRIVATE }}
           passphrase: ${{ secrets.HYP_BOT_GPG_PASSWORD }}
           git-user-signingkey: true
           git-commit-gpgsign: true
-          git_config_global: true
-          git_tag_gpgsign: true
+          git-config-global: true
+          git-tag-gpgsign: true
```

Comment:
? why this change?
The ones above with `-` is what seems wrong
What documentation are use checking because I'm checking the examples in https://github.com/crazy-max/ghaction-import-gpg

Reply:
@FabioPinheiro , afaik, the names are different if we use a newer version of the gpg github action, so it's acceptable.